resources/logback.xml

```diff
@@ -18,7 +18,7 @@
 
     <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
       <!-- rollover daily -->
-      <fileNamePattern>logs/hecuba-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+      <fileNamePattern>logs/nhs-application-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
```

Comment:
Would be good to add a syslog endpoing as well so we can ship it to ELK. @sw1nn has an example in another repo.